code/src/App.js

```diff
@@ -1,9 +1,67 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+
+import MessageInput from './components/MessageInput';
+import MessageList from './components/MessageList';
+import Loader from './components/Loader';
+import { MESSAGES_URL } from './urls';
+
+import './components/Styles.scss';
 
 export const App = () => {
+  const [messages, setMessages] = useState([]);
+  const [isLoading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetchMessages();
+    setInterval(fetchMessages, 5000);
```

Comment:
Neat!

Reply:
I do not consider this as a good idea.
I understand that you probably want to update the list each 5 seconds to provide user with the newest set of data, but we shouldn't do it like this. There is a technique called Subscription which is being used for this kind of communication between front-end and back-end to detect when BE data was updated, but it's clearly based on something far more complicated than doing fetch request each 5 seconds.
For now let's delete this line of code.
Nonetheless, I appreciate your intuition, it's great that you are aware of the fact that data on BE side can be updated without FE concern :)